conftest.py

```diff
@@ -1,32 +0,0 @@
-import sys
-
-import pytest
-
-
-def pytest_addoption(parser):
-    parser.addoption(
-        "--skip-formatting",
```

Comment:
This formatting flag is also used for our doc tests; I think we should keep it but with a rename (I generally don't run doc tests locally until I'm nearing the end)